lib/vet360/models/address.rb

```diff
@@ -28,6 +36,50 @@ class Address < Base
       attribute :zip_code_suffix, String
 
       validates :source_date, presence: true
+
+      validates(
+        :address_pou,
+        presence: true,
+        inclusion: { in: ADDRESS_POUS }
+      )
+
+      validates(
+        :address_type,
+        presence: true,
+        inclusion: { in: ADDRESS_TYPES }
+      )
+
+      def details
+        case address_type
+        when DOMESTIC
+          domestic_address
+        when INTERNATIONAL
+          international_address
+        when MILITARY
+          military_address
+        end
+      end
+
+      def domestic_address
+        {
+          id: id,
+          address_line_1: address_line_1,
+          address_line_2: address_line_2,
+          address_line_3: address_line_3,
+          city: city,
+          state_abbr: state_abbr,
+          zip_code: zip_code,
+          zip_code_suffix: zip_code_suffix
+        }
+      end
```

Comment:
@bshyong @ncksllvn - anything else you'd like to see returned for a "domestic" address?

Reply:
Looks good. maybe in the future we will want to surface the last_updated/effective_start_date so that we can prompt users to check that information is still accurate, but I don't think we have a need for that now